.github/workflows/ci.yml

```diff
@@ -37,6 +37,11 @@ jobs:
           - os: "windows-latest"
             ghc: "9.8"
 
+          # Cannot build on Windows due to problem with dependency on package
+          # "Win32"
```

Comment:
I think if you'd run `cabal outdated` on Windows you'd see the out-of-bounds message for `Win32`.